From cd9e6d15ff2d4d7fa50288083056d649d5123ee1 Mon Sep 17 00:00:00 2001
From: Tim Gover <tim.gover@raspberrypi.org>
Date: Wed, 5 Sep 2018 22:09:28 +0100
Subject: [PATCH 180/198] rpi-simple-soundcard: Use nicer driver name
 "RPi-simple"

commit 96815f74d4770051048df4d9fce9d773ddddcf94 from
https://github.com/raspberrypi/linux.git

Rename the driver from "RPI simple soundcard" to "RPi-simple" so that
the driver name won't be mangled allowing to be used unaltered as the
card conf filename.
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 sound/soc/bcm/rpi-simple-soundcard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/bcm/rpi-simple-soundcard.c b/sound/soc/bcm/rpi-simple-soundcard.c
index 811345d6a61d..7fa92e29502a 100644
--- a/sound/soc/bcm/rpi-simple-soundcard.c
+++ b/sound/soc/bcm/rpi-simple-soundcard.c
@@ -202,7 +202,7 @@ static const struct of_device_id snd_rpi_simple_of_match[] = {
 };
 
 static struct snd_soc_card snd_rpi_simple = {
-	.driver_name  = "RPI simple soundcard",
+	.driver_name  = "RPi-simple",
 	.owner        = THIS_MODULE,
 	.dai_link     = NULL,
 	.num_links    = 1, /* Only a single DAI supported at the moment */
-- 
2.17.1

